lib/SiftResponse.php

```diff
@@ -13,6 +13,11 @@ public function __construct($result, $httpStatusCode, $request) {
         $this->apiStatus = null;
         $this->apiErrorMessage = null;
 
+        // Check for errors and bad httpStatusCode
```

Comment:
Ah! Yeah, once the 999 is removed, it's cleaner to get rid of:

```
if($result->error && $httpStatusCode == 999) {
  $this->apiErrorMessage = $result->error
}
```
and then just add in instead:

`        $this->apiErrorMessage = $result->error
`